soccer/gameplay/tactics/find_pass_point.py

```diff
@@ -0,0 +1,24 @@
+import robocup
+import constants
+from scipy.optimize import minimize
+
+## Function that optimizes cost function for passing
+# uses scipy minimize using the BFGS method
+def findPassPoint(heuristic) -> robocup.Point:
```

Comment:
It should be `find_pass_point(heuristic)`, and should probably be a higher-order function. Weren't we going to have `CoordinatedPass` accept a heuristic function rather than a single target point, so we could later extend it to optimize continuously? Am I mistaken there?

Reply:
So we've gone back and forth if we want this to be in coordinated pass or not, I could put it in coordinated pass if we wanted

Reply:
Also the function isn't really specific to passing, like you could have a cost function for positioning or shooting that could use the same function, so we might not wanna constrain it to passing.